src/test/java/org/fundacionjala/coding/german/SpinWordTest.java

```diff
@@ -0,0 +1,58 @@
+package org.fundacionjala.coding.german;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * Created by German on 8/29/2017.
+ */
+public class SpinWordTest {
+    /**
+     * Verify spinWords method.
+     */
+    @Test
+    public void testSpinWordMethodTwoReverseWords() {
+        SpinWords spinWord = new SpinWords();
```

Comment:
You are repeating this code in all your methods
please move it to a @Before method.